src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao = new DriverDaoImpl();
```

Comment:
```suggestion
    @Inject
    private DriverDao driverDao;
```
do we need to initialize fields marked with `@Inject` annotation?